tgui/packages/tgui/interfaces/Playtime.tsx

```diff
@@ -0,0 +1,107 @@
+import { classes } from 'common/react';
+import { useBackend, useLocalState } from '../backend';
+import { Tabs } from '../components';
+import { Table, TableCell, TableRow } from '../components/Table';
+import { Window } from '../layouts';
+
+interface PlaytimeRecord {
+  job: string;
+  playtime: number;
+  bgcolor: string;
+  textcolor: string;
+  icondisplay: string | undefined;
+}
+
+interface PlaytimeData {
+  stored_human_playtime: PlaytimeRecord[];
+  stored_xeno_playtime: PlaytimeRecord[];
+  stored_other_playtime: PlaytimeRecord[];
+}
+
+const PlaytimeRow = (props: { data: PlaytimeRecord }, context) => {
+  return (
+    <>
+      <TableCell className="AwardCell">
+        {props.data.icondisplay && (
+          <span
+            className={classes([
+              'AwardIcon',
+              'playtimerank32x32',
+              props.data.icondisplay,
+            ])}
+          />
+        )}
+      </TableCell>
+      <TableCell>
+        <span className="LabelSpan">{props.data.job}</span>
+      </TableCell>
+      <TableCell>
+        <span className="TimeSpan">{props.data.playtime.toFixed(1)} hr</span>
+      </TableCell>
+    </>
+  );
+};
+
+const PlaytimeTable = (props: { data: PlaytimeRecord[] }, context) => {
+  return (
+    <Table>
+      {props.data
+        .slice(1)
```

Comment:
```suggestion
        .slice(props.data.length > 1 ? 1 : 0)
```
this allows for the observer data to show up